src/pages/About/index.tsx

```diff
@@ -134,12 +135,42 @@ const Thumbnail = styled.img`
   width: 100%;
 `
 
-const SocialRow = styled.div`
-  display: flex;
+const FooterLinks = styled.div`
+  display: grid;
+  grid-template-columns: 1fr;
   gap: 24px;
 
-  & > * {
-    flex: 1;
+  @media screen and (min-width: ${BREAKPOINTS.sm}px) {
+    grid-template-columns: 1fr 1fr;
+  }
+
+  @media screen and (min-width: ${BREAKPOINTS.lg}px) {
+    grid-template-columns: 1fr 1fr 1fr 1fr;
+  }
+`
+
+const FooterLink = styled.a`
+  display: flex;
+  align-items: center;
+  gap: 16px;
+  padding: 16px;
+  color: ${({ theme }) => theme.textPrimary};
+  border: 1px solid ${({ theme }) => theme.backgroundOutline};
```

Comment:
Have you tried using a ButtonOutline or similar style from the existing collection of components?